pkgs/development/compilers/crystal/default.nix

```diff
@@ -32,6 +32,7 @@ let
     x86_64-linux = "linux-x86_64";
     i686-linux = "linux-i686";
     x86_64-darwin = "darwin-x86_64";
+    aarch64-linux = "aarch64-linux";
```

Comment:
Minor nitpick: Judging by the others, shouldn't it be `aarch64-linux = "linux-aarch64";`?

Reply:
This one comes from alpine which is following a different convention by the looks of it.

Reply:
Yes, but the value isn't used, if the alpine binary is fetched.

Reply:
- Changed the value to be `linux-aarch64` to look a little less weird.
- Changed the url function to compare against `archs.aarch64-linux` instead of the string